### DISCUSSION TITLE
Welcome to home-cinema-toolkit Discussions!

### DISCUSSION THREAD

**Discussion by luluhoc** [luluhoc/home-cinema-toolkit#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dny238** [comment]
Can I see a log of what it's doing to tell if it's working? I'm searching for 6, Action and not getting anything back. 
A log view would help me understand what's going on... Any number of API's or URL's could be wrong or broken, I suppose.

**Reply by luluhoc** [reply]
You can see the logs in docker container logs. I will add logs to the interface soon.
